src/diffusers/models/lora.py

```diff
@@ -89,6 +96,52 @@ def __init__(self, *args, lora_layer: Optional[LoRAConv2dLayer] = None, **kwargs
     def set_lora_layer(self, lora_layer: Optional[LoRAConv2dLayer]):
         self.lora_layer = lora_layer
 
+    def _fuse_lora(self):
+        if self.lora_layer is None:
+            warnings.warn("Calling fuse_lora() is not supported. It will be a no-op.", RuntimeWarning)
```

Comment:
```suggestion
            logger.warn("Calling fuse_lora() is not supported. It will be a no-op.")
```

Reply:
Decide to remove the warning completely as it might lead to more confusions.